assignment/index.html

```diff
@@ -65,7 +65,31 @@
       var jsonToCsv = function(json) { console.log(json); };
 
 
-      var addMarkers = function(map) {};
+      var addMarkers = function(map) {
+        for(var obj in healthCenters){
+
+          var lat = healthCenters[obj].LAT
```

Comment:
You could have saved yourself the creation of this variable and just added  the `healthCenters[obj].LAT` directly in line 76 in place of `lat`

Not a big deal. But it saves a little code.